raisinbread/RB_files/RB_test_battery.sql

```diff
@@ -27,12 +27,12 @@ INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Acti
 INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (154,'mri_parameter_form',1,2147483647,'Y','Visit',3,'V6',NULL,NULL,NULL);
 INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (155,'mri_parameter_form',1,2147483647,'Y','Visit',4,'V6',NULL,NULL,NULL);
 INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (156,'medical_history',1,2147483647,'Y','Visit',NULL,'V1',NULL,NULL,NULL);
-INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (157,'medical_history',1,2147483647,'Y','Visit',NULL,'V2',NULL,NULL,NULL);
+INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (157,'medical_history',1,2147483647,'N','Visit',NULL,'V2',NULL,NULL,NULL);
```

Comment:
This insert allows testing that the instrument is not populated when the Active field is set to N regardless of the other fields of that row.